databases/business-sync/fragment-shoebox-item.js

```diff
@@ -62,6 +62,53 @@
           }
         }
       }
+    },
+    annotations: {
+      type: 'array',
+      required: false,
+      arrayElementsValidator: {
+        type: 'object',
+        required: true,
+        propertyValidators: {
+          // indicates what type of annotation this is
+          // Only embedded is supported to start
+          type: {
+            type: 'enum',
+            immutable: true,
+            required: true,
+            predefinedValues: [ 'embedded' ]
+          },
+          // a simple tag describing what type data this annotation is
+          // example:  'user-metadata', 'ocr-analysis', 'model-XYZ-classification'
+          dataType: {
+            type: 'enum',
+            immutable: true,
+            required: true,
+            predefinedValues: [ 'user-metadata' ]
+          },
+          // payload, pretty much unrestricted at this point
+          data: {
+            type: 'object',
+            required: true
+          },
+          // timestamp recording when this annotation was last updated
+          lastModified: {
+            type: 'datetime',
+            required: true
+          },
+          // ID of the annotating user
+          annotatingUser: {
```

Comment:
If we allow untracked edits/changes to existing annotations, different users could be making different edits.  In this case, it's not clear what should be stored in the user Id field.  Should it be the user Id of the most recent change?  Perhaps `lastModified` should instead be an array of timestamp/userId tuples, even if we don't track the actual data changes?

Reply:
Having that kind of history has proven useful. That being said, if we're not able to tell what the changeset was I wonder if it becomes less useful. However, there is something to overwriting the `lastModified` that doesn't feel quite right. I could be convinced of it though based on YAGNI. However, if having it be an array of tuples isn't a lot of work it seems like a reasonable approach given what we know at this time.